stdlib/Distributed/src/cluster.jl

```diff
@@ -605,7 +608,12 @@ function launch_additional(np::Integer, cmd::Cmd)
     end
 
     for (i,io) in enumerate(io_objs)
-        (host, port) = read_worker_host_port(io)
+        try
+            (host, port) = read_worker_host_port(io)
+        catch e
+            kill(io)
```

Comment:
`kill` needs either a Process object or individual cluster managers need to implement it. I suggest that we throw a `TimedOutException` and handle it specifically in the cluster manager. I think you will need to call a command specific to the particular cluster manager, i.e., specific to LSF, SLURM, etc...

Reply:
i've moved the `kill` to  `create_worker()`, which has all the needed info to call the method in ClusterManagers.